rasterio/rio/info.py

```diff
@@ -72,8 +72,12 @@ def env(ctx, key):
               help="Output extra information.")
 @click.option('--bidx', type=int, default=1,
               help="Input file band index (default: 1).")
+@click.option('--masked/--raw',
```

Comment:
The default here is `True` (opposite of this PR), but the context of this mode should provide representative statistics info.